scripts/gen_openapi.sh

```diff
@@ -36,7 +36,9 @@ vault auth enable github
 vault auth enable jwt
 vault auth enable kubernetes
 vault auth enable ldap
+vault auth enable oidc
```

Comment:
Wondering if we need this one since its just a name alias for jwt?

Reply:
Ah! OK, I will strike it.